dev-python/ruamel-yaml/ruamel-yaml-0.16.5.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{5,6,7,8} pypy3 )
+
+inherit distutils-r1 vcs-snapshot
+
+DESCRIPTION="YAML parser/emitter that supports roundtrip comment preservation"
+HOMEPAGE="https://pypi.org/project/ruamel.yaml/ https://bitbucket.org/ruamel/yaml"
+MY_PN="${PN//-/.}"
+SRC_URI="https://bitbucket.org/${MY_PN/.//}/get/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~x86"
+IUSE="test +clib"
+
+BDEPEND="
+	dev-python/setuptools[${PYTHON_USEDEP}]
+	clib? ( dev-python/ruamel-yaml-clib[${PYTHON_USEDEP}] )
```

Comment:
Would be great to have the real change separated from ebuild file cloning into standalone commit.
Without that, I recognise that this addition of a USE flag is the most important change.
Could you please elaborate the reasoning behind this change, in commit message please?

I don't see any change to configure and compile stages, so we must be ending up with one of
* USE flag for optional runtime dependency, which is bad;
* package's dependency discovery doing its uncontrolled automagic, which is bad.

Reply:
Dephell supports both ruamel.yaml and PyYAML. So, if installing the first one is tricky, just use the second one.